tests/astype_test.py

```diff
@@ -45,3 +45,11 @@ def test_astype_empty(df_factory):
     df = df_factory(x=[1, 2, 3])
     df = df[df.x<0]
     assert len(df.x.as_numpy().values) == 0
+
+
+def test_astype_timedelta(df_factory):
+    x = [23423, -242, 34656]
+    x_result = np.array([23423,  -242, 34656], dtype='timedelta64[s]')
+    df = df_factory(x=x)
+    df['x_expected'] = df.x.astype('timedelta64[s]')
```

Comment:
this does not test what you want I think, because x_result already has the dtype you want.

Reply:
ok, I was confused, if you reuse x I think it is more clear.

Reply:
It works, but the variable names are the wrong way around :S. I changed them to be what one would expect.